onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -101,12 +101,7 @@ OpenVINOExecutionProvider::GetCapability(const GraphViewer& graph_viewer, const
     std::cout << "In the OpenVINO EP" << std::endl;
   }
   openvino_ep::BackendManager::GetGlobalContext().onnx_model_name = graph_viewer.Name();
-#ifdef _WIN32
-  std::wstring onnx_path = graph_viewer.ModelPath().ToPathString();
-  openvino_ep::BackendManager::GetGlobalContext().onnx_model_path_name = std::string(onnx_path.begin(), onnx_path.end());
-#else
-  openvino_ep::BackendManager::GetGlobalContext().onnx_model_path_name = graph_viewer.ModelPath().ToPathString();
-#endif
+  openvino_ep::BackendManager::GetGlobalContext().onnx_model_path_name = ToUTF8String(graph_viewer.ModelPath().ToPathString());
```

Comment:
I think you should change the type of "onnx_model_path_name" to std::basic_string<ORTCHAR_T> instead. On Windows, path strings rarely are UTF-8. I guess it wouldn't work.

Reply:
Hello @snnn! I'm not a developer of OpenVino EP. I guess that the developers expect that "onnx_model_path_name" field is string for correct its processing further. As you see from previous code wstring (model path) is still converted to std::string, but by wrong way (I try to fix this moment). They could change type of "onnx_model_path_name" field to PathString but 1. they did not do it, 2. it requires more changings in the code to correct processing of the field on different OS.

Reply:
Hello @snnn! What we will do with this PR? We can ask somebody from OpenVivo to review this if you know anybody or close it. For me it does not influence on my work, it is by product of the work.